tools/src/test/java/org/apache/kafka/tools/ToolsTestUtils.java

```diff
@@ -42,7 +42,7 @@
 
 public class ToolsTestUtils {
     /** @see TestInfoUtils#TestWithParameterizedQuorumName()  */
-    public static final String TEST_WITH_PARAMETERIZED_QUORUM_NAME = "{displayName}.quorum={0}";
+    public static final String TEST_WITH_PARAMETERIZED_QUORUM_NAME = "{displayName}.{argumentsWithNames}";
```

Comment:
Changing this one to be aligned with https://github.com/apache/kafka/pull/14687.